docs/getting_started/rules.md

```diff
@@ -113,28 +115,12 @@ class TestMyNewRule(BaseRuleTestCase):
 As you can see `test_file_negative()` in this unit test makes specific use of a CloudFormation template from the Fixtures folder. It's important to provide examples of templates which pass your new rule, and also templates which generate the expected warning/error.
 
 #### Running the Tests
-The unit tests (and `pylint` for linting the Python code) are processed by [`Tox`](https://tox.readthedocs.io/en/latest/).
-
-Make sure Tox is installed and then run Tox by just calling it:
 
-```bash
-# Install Tox
-$ pip install tox
+Please see detailed instructions [here](/CONTRIBUTING.md).
```

Comment:
```suggestion
Please see detailed instructions [here](../../CONTRIBUTING.md).
```

not sure if local markdown links are even a thing, but the root directory would be differently locally than online